governance/votes/Executive vote - July 2, 2021.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Template - [Executive Vote] Rates Changes, July Core Unit Budgets, and PSM DC-IAM - July 2, 2021
+summary: Vote for Rates Changes, July Core Unit Budgets, and PSM DC-IAM
+date: 2021-07-02T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Rates Changes, July Core Unit Budgets, and PSM DC-IAM - July 2, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+**Core Unit Budget Distributions - July**  
+DAI will be distributed to the Core Units as specified within their ratified budget proposals.
+
+| Core Unit | Distribution |
```

Comment:
I'm assuming SES will be in next weeks

Reply:
Next months, apparently they're fine with that.